addons/survey/views/survey_question_views.xml

```diff
@@ -142,15 +148,22 @@
                                     </group>
                                 </div>
                             </div>
-                            <group attrs="{'invisible': [('question_type', 'not in', ['simple_choice', 'multiple_choice', 'matrix'])]}">
+                            <group attrs="{'invisible': [('question_type', 'not in', ['simple_choice', 'multiple_choice'])]}">
+                                <field name="suggested_answer_ids" context="{'default_question_id': active_id}" nolabel="1">
```

Comment:
Also, why do we have the field "suggested_answer_ids" twice now?
It's usually not a good idea to have the same field twice in a view.
Wasn't it not working properly with the column_invisible checks?